test/integration/query.test.ts

```diff
@@ -204,6 +204,16 @@ describe('integration tests', () => {
     expect(teams).toHaveLength(0);
   });
 
+  test('filter is empty', async () => {
```

Comment:
I saw one of the issues was for `.filter({})` should we add a quick test for that too?

Reply:
Also included in that test! It's actually kind of empty (we don't send any filters to the API)